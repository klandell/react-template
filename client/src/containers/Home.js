import React from 'react';

export default class Home extends React.Component {
    render() {
        return (
            <div className="home-wrapper">
                <h1>Home</h1>
            </div>
        );
    }
}
