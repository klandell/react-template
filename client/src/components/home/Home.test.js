import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from './Home';

test('Home component should render correctly.', () => {
    const component = shallow(<Home />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});
