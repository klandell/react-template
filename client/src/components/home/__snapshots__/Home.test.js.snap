// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render correctly. 1`] = `
<div
  className="home"
>
  <h1>
    Home
  </h1>
</div>
`;
